smartsim/entity/model.py

```diff
@@ -46,6 +46,24 @@
 logger = get_logger(__name__)
 
 
+def _read_model_parameters(params_dict: t.Dict[str, t.Any]) -> t.Dict[str, str]:
+    """Convert the values in a params dict to strings
+    :raises TypeError: if params are of the wrong type
+    :return: param dictionary with values and keys cast as strings
+    """
+    param_names: t.List[str] = []
+    parameters: t.List[t.List[str]] = []
+    for name, val in params_dict.items():
+        param_names.append(name)
+        if isinstance(val, (int, str)):
+            parameters.append(str(val))
+        else:
+            raise TypeError(
+                "Incorrect type for ensemble parameters\n" + "Must be int, or string."
```

Comment:
`ensemble` -> `model`

`int, or string` to `numeric value or string`

Reply:
changed